File: src/main/java/io/github/oleksivio/telegram/bot/api/model/method/game/SetOwnGameScore.java
package io.github.oleksivio.telegram.bot.api.model.method.game;

import io.github.oleksivio.telegram.bot.api.model.objects.std.Message;
import io.github.oleksivio.telegram.bot.core.controller.network.ActionNetworker;
import io.github.oleksivio.telegram.bot.core.model.responses.CommonResponse;
import io.github.oleksivio.telegram.bot.core.model.responses.MessageResponse;
import io.github.oleksivio.telegram.bot.core.model.method.game.SetGameScore;

public class SetOwnGameScore extends SetGameScore<Message> {
    public SetOwnGameScore(ActionNetworker actionNetworker) {
        super(actionNetworker);
    }

    @Override
    protected Class<? extends CommonResponse<Message>> getResultWrapperClass() {
        return MessageResponse.class;
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/method/game/SetOwnGameScore.kt
package io.github.oleksivio.telegram.bot.api.model.method.game

import io.github.oleksivio.telegram.bot.api.model.objects.std.Message
import io.github.oleksivio.telegram.bot.core.controller.network.ActionNetworker
import io.github.oleksivio.telegram.bot.core.model.MessageResponse
import io.github.oleksivio.telegram.bot.core.model.method.game.SetGameScore

class SetOwnGameScore(actionNetworker: ActionNetworker) : SetGameScore<Message>(actionNetworker) {

    override val resultWrapperClass = MessageResponse::class
}
